documentation/tut/toy-example-expanded.rst

```diff
@@ -87,59 +93,32 @@ This will have an effect on the available headroom for the battery, given the ``
 
             $ flexmeasures add schedule \
                 --sensor 2 \
-                --start ${TOMORROW}T07:00+01:00 \
+                --start ${TOMORROW}T07:00+00:00 \
                 --duration PT12H \
                 --soc-at-start 50% \
-                --flex-model '{"roundtrip-efficiency": "90%"}' \
                 --flex-context '{"inflexible-device-sensors": [3]}'
+                --flex-model '{"soc-min": "50 kWh"}' \
             New schedule is stored.
-
+        
     .. tab:: API
 
-        Example call: `[POST] http://localhost:5000/api/v3_0/assets/2/schedules/trigger <../api/v3_0.html#post--api-v3_0-assets-(id)-schedules-trigger>`_ (update the start date to tomorrow):
+        Example call: `[POST] http://localhost:5000/api/v3_0/sensors/2/schedules/trigger <../api/v3_0.html#post--api-v3_0-sensors-(id)-schedules-trigger>`_ (update the start date to tomorrow):
```

Comment:
Don't you also prefer first-time users to use the asset trigger endpoint straightaway, instead of using the sensor trigger endpoint?